examples/cli.py

```diff
@@ -146,6 +147,8 @@ def Main() -> None:
                 ('--filter', 'Query filter.', None)
             ])
   AddParser('gcp', gcp_subparsers, 'listlogs', 'List GCP logs for a project.')
+  AddParser('gcp', gcp_subparsers, 'listservices',
+            'List active APIS for a project.')
```

Comment:
```suggestion
            'List active APIs for a project.')
```

If you're listing APIs, shouldn't this be called `listapis`? Please adjust the command name or the help text